modules/application-health-about.adoc

```diff
@@ -51,8 +51,8 @@ initialization is complete.
 You can configure several fields to control the behavior of a probe:
 
 * `initialDelaySeconds`: The time, in seconds, after the container starts before the probe can be scheduled. The default is 0.
-* `periodSeconds`: The delay, in seconds, between performing probes. The default is `10`.
-* `timeoutSeconds`: The number of seconds of inactivity after which the probe times out and the container is assumed to have failed. The default is `1`.
+* `periodSeconds`: The delay, in seconds, between performing probes. The default is `10`. Must be greater than `timeoutSeconds`. 
```

Comment:
This is clear as is, so feel free to ignore: 

For "Must be greater/less than..." references, would it be helpful to the user to clarify, "The value must be greater/less than..."?